.github/workflows/build.yaml

```diff
@@ -14,19 +17,19 @@ on:
 jobs:
 
   build:
-    runs-on: self-hosted
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - name: Login to DockerHub
       run: echo ${{ secrets.docker_password }} | docker login --username ${{ secrets.docker_username }} --password-stdin
 
     # Build images
     - name: INFO BUILD
-      run: echo ${{ github.event.inputs.image }} ${{ github.event.inputs.version }}
+      run: echo ${{ github.event.inputs.branch }} ${{ github.event.inputs.image }} ${{ github.event.inputs.version }}
```

Comment:
I think this line 28 was meant to use `github.event.inputs.path` instead of `github.event.inputs.branch`, no?